content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -1,50 +1,62 @@
 ---
 title: Parallel Processing using Expansions
-content_template: templates/concept
+content_template: templates/task
 weight: 20
 ---
 
 {{% capture overview %}}
 
-In this example, we will run multiple Kubernetes Jobs created from
-a common template.  You may want to be familiar with the basic,
-non-parallel, use of [Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) first.
+In this example, you will run multiple {{< glossary_tooltip text="Jobs" term_id="job" >}}
+based a common template. You can use this approach to process batches of work in
```

Comment:
@sftim, couple of nits/how about this wording to get rid of _will_:
This example shows/explains how to run multiple Jobs [tooltip stuff] based on a common template.
You can use this approach ...

Reply:
Reworded, inspired by above suggestion